### PR TITLE
Improve the host details view E2E to cover more data

### PR DESCRIPTION
**Pull request by arbulu89**
Increase some test coverage in the Host details view to check more cloud details and sles subscription data.

### CODE REVIEW

**Review by dottorblaster — APPROVED** [review]

File: test/e2e/cypress/fixtures/host-details/selected_host [hunk 1]
```diff
@@ -1,9 +1,46 @@
 export const selectedHost = {
```
Comment (nelsonkopliku) [inline test/e2e/cypress/fixtures/host-details/selected_host:1]: I believe we are missing the file extension here `.js`

**Review by nelsonkopliku — CHANGES_REQUESTED** [review]
LGTM, except for a missing file extension (Yes I know you found it already there without extension :smile: )

File: test/e2e/cypress/fixtures/host-details/selected_host (hunk 1 above)
Comment (arbulu89) [reply test/e2e/cypress/fixtures/host-details/selected_host]: Good catch! Fixed

**Review by nelsonkopliku — APPROVED** [review]
:shipit:

**Pull request merged**